packages/cli/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hubspot/cli",
-  "version": "5.1.1",
+  "version": "5.1.2",
```

Comment:
You shouldn't need to manually bump this. lerna will handle it for you when you run the release command.

Reply:
Took it back down